samples/guide/src/main/java/okhttp3/recipes/CheckHandshake.java

```diff
@@ -59,6 +59,10 @@ public void run() throws Exception {
   }
 
   public static void main(String... args) throws Exception {
-    new CheckHandshake().run();
+    try {
+      new CheckHandshake().run();
+    } catch (IOException ioe) {
+      // expected
```

Comment:
nit: maybe change the variable name from `ioe` to `ignored` or `expected` and remove the comment? :)